strider/fetcher.py

```diff
@@ -298,6 +261,7 @@ async def processor(request, logger: logging.Logger = None):
     async def setup(
             self,
             qgraph: dict,
+            use_cache: bool = True,
```

Comment:
Maybe we can make this a part of the settings object?

Reply:
Yeah, it looks like there's currently no external way of setting this. I think it can either go in the settings object, or we make it a param on the `query` and `asyncquery` endpoints. Do you have a preference? I think I'm leaning towards putting it in settings. I don't think users would ever want to _not_ cache.

Reply:
I agree that the settings object is the way to go.

Reply:
Moved in 066b06c